tests/baselines/reference/tsbuild/outfile-concat/initial-build/multiple-emitHelpers-in-different-projects.js

```diff
@@ -304,26 +304,6 @@ sourceFile:../second/second_part2.ts
 >>>//# sourceMappingURL=second-output.d.ts.map
 
 //// [/src/2/second-output.js]
-var __read = (this && this.__read) || function (o, n) {

```

Comment:
can you please modify the test case to use multiple emit helpers given this one removes it completely.